neuralmonkey/dataset.py

```diff
@@ -95,6 +95,84 @@ def __init__(self,
 # pylint: enable=too-few-public-methods
 
 
+def _bucket_boundaries(max_length, min_length=8, length_bucket_step=1.1):
```

Comment:
Proč je tohle ve dvou různých pull requestech?

Reply:
Tenhle PR navazuje na ten druhy. Nevim, daji se nekde tyhle zavislosti explicitne znacit?

Reply:
Můžeš říct, že to nechceš mergovat do masteru, ale do jiný branche. pak se zobrazí jen změny relevantní pro tenhle PR. Po zamergování tý base branche pak můžeš u týhle změnit target na master  a rebasovat a changelist bude vypadat stejně.